docs/ApplyConfig.md

```diff
@@ -69,7 +69,20 @@ If you would rather use the filevault-package-maven-plugin for building the pack
 </plugin>
 ```
 
-The `*.yaml` files are installed directly from the package content and respect the [run mode semantics](Configuration.md). Otherwise there is no limitation, so the YAML files will be picked up from anywhere in the package (as long as the parent node does not contain a `.` followed by one or multiple not matching run modes).
+The `*.yaml` files are installed directly from the package content and respect the [run mode semantics](Configuration.md). Otherwise there is no limitation, so the YAML files will be picked up from anywhere in the package (as long as the parent node does not contain a `.` followed by one or multiple not matching run modes). If you wish to limit where the YAML files are looked upon you can add a property `actool.hook.config.paths` in your package with a regular expression to match the desired paths:
+
+```
+<plugin>
+    <groupId>org.apache.jackrabbit</groupId>
+    <artifactId>filevault-package-maven-plugin</artifactId>
+    <configuration>
+        <properties>
+            <installhook.actool.class>biz.netcentric.cq.tools.actool.installhook.AcToolInstallHook</installhook.actool.class>
+            <actool.hook.config.paths>/apps/myapp/acl.*</actool.hook.config.paths>
```

Comment:
How are multiple values given?
Please rather name it `actool.hook.config.path.patterns` to make it clear that this is a regex!